c-sharp-server/c-sharp-client/Task.cs

```diff
@@ -1,54 +1,43 @@
 ﻿using System;
 using System.Collections.Generic;
-using DittoSDK;
+using System.Text.Json;
 
 namespace Program
 {
-
     public struct Task
     {
-        string _id;
-        string body;
-        bool isCompleted;
+        public string _id { get; set; }
+        public string body { get; set; }
+        public bool isCompleted { get; set; }
+        public bool isDeleted { get; set; }
```

Comment:
Naming convention for Properties is PascalCase

```suggestion
        public string Id { get; set; }
        public string Body { get; set; }
        public bool IsCompleted { get; set; }
        public bool IsDeleted { get; set; }
```

Reply:
Ditto Documents typically have `_id` and lowercase at the beginning for keys, and this sample uses `JsonSerializer.Deserialize` to deserialize objects. I imagine having different naming patterns for apps would complicate this sample, so I'll leave this as it is for now.